### PR TITLE
Expose connector through gateway proxy

### PR DESCRIPTION
**Pull request by crabtree**
**Description**

Changes proposed in this pull request:

- Host connector through the gateway proxy

**Related issue(s)**

See also #18

### CODE REVIEW

File: chart/compass/values.yaml [hunk 1]
```diff
@@ -2,12 +2,15 @@ global:
   images:
     containerRegistry:
       path: eu.gcr.io/kyma-project/incubator
+    connector:
+      dir: pr/
+      version: "PR-32"
```
Comment (pkosiec) [inline chart/compass/values.yaml]: It doesn't work - you have to modify Connector (parametrize Playground API endpoint), similarly to Director. Just copy `main.go` from `director`, adjust it if necessary and it'll work.

File: chart/compass/values.yaml [hunk 2]
```diff
@@ -2,12 +2,15 @@ global:
...
     director:
       dir: pr/
       version: "PR-39"
```
Comment (pkosiec) [inline chart/compass/values.yaml]: Please change the image of Director to `PR-41`. Apparently I forgot to change it 😞

Comment (crabtree) [reply chart/compass/values.yaml]: Done

File: chart/compass/values.yaml (hunk 1 above)
Comment (crabtree) [reply chart/compass/values.yaml]: Fixed

**Pull request merged**